chapters/methodology.tex

```diff
@@ -1,6 +1,12 @@
 \section{Methodology} \label{sec:methodology}
 % detailing the approaches that were used in your experiments; this may also include implementation details in relation to the mobile architecture used
-According to \cite{sample_ref}, lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
+The development process included some beneficial agile development methods.
+The project team used daily stand-ups to share their individual work progress and inform about current difficulties.
+A shortly conclusion meeting at the end of every day was hold to summarize the work effort and supported a productive communication.
+A kanban board added organization clarity and displayed the responsibility for every identified task.
+At the beginning of the week, we started with a retrospection, followed by identification the mile stone for the week.

```

Comment:
Maybe write something like:

"At the beginning of every week, a retrospection / retrospective (your preference) of the past week was conducted, followed by identification of the mile stone for the week."

Reply:
Actually this is an even better alternative 😄